src/javascript/flow/codeGeneration.ts

```diff
@@ -178,6 +179,19 @@ export class FlowAPIGenerator extends FlowGenerator {
 
     this.printer.enqueue(exportedTypeAlias);
     this.scopeStackPop();
+
+    // Generate the variables interface if the operation has any variables
+    if (variables.length > 0) {
```

Comment:
Suggestion: Is there a good reason to skip this type definition here, versus creating an empty object type or null type? Omitting a definition here prevents other code generators from consistently importing `*Query, *Variables` types for each operation without having to do the same kind of check.

It would be helpful in those cases to have a type generated for consistency, like:
```ts
export type SomeStaticQueryVariables = {};
```

Reply:
(Added a PR to implement as a compiler option: #350)